src/components/History/Timeline/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import HistoryTile from './historytile';
+
+export default class Timeline extends React.Component {
+	render() {
+		return (
+			<div id="timeline">
+				{ this.props.history.map(history => <HistoryTile key={history.title} history={history} /> ) }
```

Comment:
this is a little picky, but remove one space before `)` like this:
`{ this.props.history.map(history => <HistoryTile key={history.title} history={history} />) }`

when i add a linter, things like this should be picked up on automatically